### DISCUSSION TITLE
Can Conv-LoRa weights be merged into the original model?

### DISCUSSION THREAD

**Discussion by hiyyg** [autogluon/autogluon#4572] [Q&A] [answered]
@Harry-zzh It seems the merging operation is not implemented for Conv-LoRa.

**Comment by Harry-zzh** [comment, marked as answer]
> @Harry-zzh It seems the merging operation is not implemented for Conv-LoRa.

Conv-LoRA can't be merged into the original model due to its interpolating operations and non-linear activations.
